src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const li = document.querySelectorAll('li');
+
+for (let i = 0; i < li.length; i++) {
+  const span = document.createElement('span');
+  const firstChild = li[i].firstChild;
+
+  if (firstChild.textContent !== undefined) {
```

Comment:
Instead of checking if 'firstChild.textContent' is not undefined, you can use a truthy check. If 'firstChild.textContent' exists, it will pass the check.

```suggestion
  if (firstChild.textContent) {
```

Reply:
if you modify function to be forEach like above you don't need this code anymore